### PR TITLE
Completed html-forms exercise - Julie Dumbrique

### PR DESCRIPTION
**Pull request by juliedumbrique**
https://user-images.githubusercontent.com/72038937/129805781-4c587c4c-0943-42bf-8758-eb48a27032a7.mov

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get" class="form-example">
+    <div class="form-example">
+      <label for="email">Email</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:10]: This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      <div><input type="email" name="email" /></div>
+      <label for="cool">Why is coding cool?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:12]: This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <div><select name="options">
+      <option value="option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select></div>
+      <label for="choose">Favorite Language (choose one)</label>
```
Comment (codingcodymiller) [inline html-forms/index.html]: This should just be a general text element such as `p`, it isn't labelling a specific input.

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    <div><radio>
+      <input type="radio" name="language" value="html" checked="checked" /> HTML
+      <input type="radio" name="language" value="css" /> CSS
+      <input type="radio" name="language" value="javascript" /> JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html]: The text for each of these radio inputs should be a `label`

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    </radio></div>
+    <label for="signup">Would you like to sign up for more information?</label>
+    <div><checkbox>
```
Comment (codingcodymiller) [inline html-forms/index.html]: There is no such element as `checkbox`, you should remove that.

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select></div>
+      <label for="choose">Favorite Language (choose one)</label>
+    <div><radio>
+      <input type="radio" name="language" value="html" checked="checked" /> HTML
+      <input type="radio" name="language" value="css" /> CSS
+      <input type="radio" name="language" value="javascript" /> JavaScript
+    </radio></div>
+    <label for="signup">Would you like to sign up for more information?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:31]: This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    <div><checkbox>
+      <input type="checkbox" name="more" value="signup" /> Sign Me Up!
+    </checkbox></div>
+    <label for="comments">Comments</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:35]: This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here!! A few notes for you:

**Comment by juliedumbrique** [comment]
https://user-images.githubusercontent.com/72038937/129829817-d95ba3c7-4207-456c-93f0-e26e99074f49.mov

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**